arches/install/arches-templates/tests/test_settings.py-tpl

```diff
@@ -0,0 +1,111 @@
+"""
+ARCHES - a program developed to inventory and manage immovable cultural heritage.
+Copyright (C) 2013 J. Paul Getty Trust and World Monuments Fund
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as
+published by the Free Software Foundation, either version 3 of the
+License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+"""
+
+from arches.settings import *
+import arches
```

Comment:
Some [additional settings](https://github.com/archesproject/arches-for-science/blob/dev/1.1.x/arches_for_science/test_settings.py) we found to be necessary in AfS to get urls to reverse correctly in tests:

- APP_NAME
- APP_ROOT
- ROOT_URLCONF

For example, removing ROOT_URLCONF and adding a breakpoint in the afs tests gives:
```py
(Pdb) reverse('deletesamplearea')
*** django.urls.exceptions.NoReverseMatch: Reverse for 'deletesamplearea' not found. 'deletesamplearea' is not a valid view function or pattern name.
```